src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs

```diff
@@ -36,11 +38,18 @@ public static object ToObject(this TypedData typedData)
                 case RpcDataType.Json:
                     return JsonConvert.DeserializeObject(typedData.Json, _datetimeSerializerSettings);
                 case RpcDataType.Http:
-                    return GrpcMessageExtensionUtilities.ConvertFromHttpMessageToExpando(typedData.Http);
+                    return GrpcMessageExtensionUtilities.ConvertFromHttpMessageToExpando(typedData.Http, sharedMemoryManager);
                 case RpcDataType.Int:
                     return typedData.Int;
                 case RpcDataType.Double:
                     return typedData.Double;
+                case RpcDataType.SharedMemoryData:
```

Comment:
Can you take a look at the proto file changes I requested?  Moving new type: memory mapped data under `parameterbinding` will simplify this code.